### ISSUE TITLE
Upgrade SES limits

### ISSUE THREAD

**Issue by jontg** [technologists-for-progress/52calls-52weeks#12]
Blocked on getting SSL cert and developing a landing page, and probably also on at least some rudimentary work flow to verify our process works.

**Comment by jontg** [comment]
We're using MailChimp (at least for V1), so closing this issue as no longer relevant.
